src/shared_modules/dbsync/src/dbsync_implementation.cpp

```diff
@@ -49,14 +49,28 @@ void DBSyncImplementation::insertBulkData(const DBSYNC_HANDLE   handle,
     ctx->m_dbEngine->bulkInsert(json.at("table"), json.at("data"));
 }
 
+static bool shouldReturnOldData(const nlohmann::json& data)
+{
+    auto it { data.find("return_old_data") };
+
+    if (it != data.end())
+    {
+        return *it;
```

Comment:
Hi @RamiroRD 

TFor RVO C++ is better only using one return.

Best regards
Hanes

Reply:
Right, I missed this one. Thanks